examples/json/src/main.rs

```diff
@@ -76,5 +74,6 @@ fn main() {
     rocket::ignite()
```

Comment:
Move this into a `rocket` function like the `forms` example does, and then call that function from `main` and from the tests as well.